### PR TITLE
Use registerMlirTranslations() in iree-translate

### PR DESCRIPTION
**Pull request by marbre**

### CODE REVIEW

File: iree/tools/BUILD [hunk 1]
```diff
@@ -111,6 +111,16 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "Translations",
+    hdrs = [
+        "init_translations.h",
+    ],
+    deps = [
+        "SPIRVTranslateRegistration",
```
Comment (GMNGeoffrey) [inline iree/tools/BUILD:120]:
```suggestion
        ""@llvm-project//mlir:SPIRVTranslateRegistration"",
```

Comment (marbre) [reply iree/tools/BUILD:120]: Obviously... Thanks for catching this!

Comment (GMNGeoffrey) [reply iree/tools/BUILD:120]: Except I've got two many quotes in my suggestion :-D

Comment (GMNGeoffrey) [reply iree/tools/BUILD:120]: Also I didn't, the CI did :-P

Comment (marbre) [reply iree/tools/BUILD]: Yeah, and I used github to submit :see_no_evil: Anyway, only needed to rewrite the commit message.

**Pull request closed without merging**